lib/sequel/extensions/newrelic_instrumentation.rb

```diff
@@ -62,7 +62,7 @@ def self.query_method_name
     # block in the sequel Plugin instrumentation. A simple ORM operation from the Plugin
     # instrumentation may trigger a number of calls to this method. We want to append
     # the statements that come from the disable_all_tracing block into this node's
-    # statement, otherwise we would end up ovewriting the sql statement with the
+    # statement, otherwise we would end up overwriting the sql statement with the
```

Comment:
Note that sometimes people mean `overriding`, but I think this is correct as corrected.

Reply:
Agreed.